modules/astcenc/config.py

```diff
@@ -1,5 +1,5 @@
 def can_build(env, platform):
-    return env.editor_build
+    return env.editor_build or env.get("angle_libs")
```

Comment:
Might warrant a comment.

We don't use the astcenc in non-editor builds as it's only used in the editor to encode textures. I assume ANGLE needs it in export templates too to re-encode some stuff on the fly?

Reply:
Yes, ANGLE always needs it (previously was part of ANGLE lib).